server/etcdserver/txn/txn.go

```diff
@@ -114,17 +121,19 @@ func DeleteRange(kv mvcc.KV, txnWrite mvcc.TxnWrite, dr *pb.DeleteRangeRequest)
 	return resp, nil
 }
 
-func Range(ctx context.Context, lg *zap.Logger, kv mvcc.KV, txnRead mvcc.TxnRead, r *pb.RangeRequest) (*pb.RangeResponse, error) {
+func Range(ctx context.Context, lg *zap.Logger, kv mvcc.KV, r *pb.RangeRequest) (*pb.RangeResponse, error) {
```

Comment:
It seems that the `Range` will never be called. It isn't a big problem.

Reply:
What do you mean? Range is called in apply.go.

Reply:
There is no a standalone Range call coming from raft (apply.go) directly.

Previously the Range may be called in a TXN (applyTxn) or called from API directly. 

After this PR, the Range will only be called from API directly.